cycledash/static/js/QueryLanguage.js

```diff
@@ -139,8 +139,13 @@ function toString(parsedQuery) {
 
   // e.g. {filters:[{type: '<', filterValue:'10', columnName:'A'}]}
   if (parsedQuery.filters) {
-    filters = filters.concat(parsedQuery.filters.map(
-        f => `${f.columnName} ${f.type} ${maybeQuote(f.filterValue)}`));
+    filters = filters.concat(parsedQuery.filters.map(f => {
+      if (f.filterValue !== null && f.filterValue !== undefined) {
```

Comment:
I changed the conditional to check the filter type, rather than the `fitlerValue`. Clearer now?